### PR TITLE
Buoyancy modified kw sst

### PR DESCRIPTION
**Pull request by gdeskos**
- Buoyancy-modified k-omega SST to model waves is introduced
- The TKE equation was modified by adding buoyancy source term that depends on the gradient of density
- Added TKE SDR initialization to the breaking waves physics class
- Two regression tests were added/but no unit tests were added

### CODE REVIEW

File: amr-wind/turbulence/RANS/KOmegaSST.cpp [hunk 1]
```diff
@@ -16,17 +16,33 @@ namespace turbulence {
...
-    pp.query("beta1", this->m_beta1);
-    pp.query("beta2", this->m_beta2);
-    pp.query("sigma_k1", this->m_sigma_k1);
-    pp.query("sigma_k2", this->m_sigma_k2);
-    pp.query("sigma_omega1", this->m_sigma_omega1);
-    pp.query("sigma_omega2", this->m_sigma_omega2);
+
+    {
+        amrex::ParmParse pp(this->model_name());
+        pp.query("is_buoyancy_modified", this->m_is_buoyancy_modified);
```
Comment (marchdf) [inline amr-wind/turbulence/RANS/KOmegaSST.cpp:22]: Would this be more explicit for the user as `include_buoyancy` ?

**Comment by marchdf** [comment]
Looks good! I just had a tiny comment. Can you also add documentation?

File: test/test_files/breaking_waves_laminar/breaking_waves_laminar.i [hunk 2]
```diff
@@ -27,8 +27,8 @@ transport.laminar_prandtl = 0.7
 transport.turbulent_prandtl = 0.3333
 turbulence.model = Laminar 
 
-incflo.physics = MultiPhase NWB 
-NWB.amplitude=0.112
+incflo.physics = MultiPhase BreakingWaves 
+BreawkingWaves.amplitude=0.112
```
Comment (mbkuhn) [inline test/test_files/breaking_waves_laminar/breaking_waves_laminar.i:31]: Looks like a typo here: Breawking -> Breaking

**Review by mbkuhn — APPROVED** [review]
Looks good!

File: amr-wind/turbulence/RANS/KOmegaSST.H [hunk 3]
```diff
@@ -85,6 +86,12 @@ protected:
     amrex::Real m_sigma_omega1{0.5};
     amrex::Real m_sigma_omega2{0.856};
     amrex::Real m_a1{0.31};
+
+    //! Buoyancy modified constants
+    bool m_include_buoyancy{false};
```
Comment (gantech) [inline amr-wind/turbulence/RANS/KOmegaSST.H:91]: I'd suggest enabling this by default only for multi-phase flows if it's an easy fix.

**Review by gantech — APPROVED** [review]
Outside of the enabling the buoyancy term by default for multi-phase flows, this looks good to me.

**Pull request merged**